### PR TITLE
[enh] add a new HTML grade for instances with analytics

### PR DESCRIPTION
**Pull request by dalf**
Related to:
* https://github.com/searxng/searx-instances/pull/20
* https://github.com/searxng/searx-space/issues/97

image

### CODE REVIEW

**Comment by unixfox** [comment]
It's great but the E letter is for `E - The instance includes file from another domain.` and that's not really the case here. It's from the same domain.
IMO the `E` letter doesn't make sense here.

File: html/main.js [hunk 1]
```diff
@@ -57,6 +58,7 @@ const HTML_GRADE_LABEL = {
     'C': 'Customized, vanilla Javascript',
     'Cjs': 'Customized, including Javascript',
     'E': 'External resources',
+    'E 👁️': 'Analytics',
```
Comment (unixfox) [inline html/main.js]:
```suggestion
    'E 👁️': 'Includes analytics software',
```

**Review by unixfox — CHANGES_REQUESTED** [review]

File: html/main.js (hunk 1 above)
Comment (unixfox) [reply html/main.js]: Or `Uses analytics software`, not very sure about the best wording.

**Comment by dalf** [comment]
image

image

**Comment by dalf** [comment]
Updated

**Review by unixfox — APPROVED** [review]

**Pull request merged**